e2e/cypress/integration/plugins/demo_plugin/webhook_spec.js

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @plugin
+
+/**
+ * Note : This test requires two demo plugin tar files under fixtures folder.
+ * Download version 0.1.0 from :
+ * https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.9.0/com.mattermost.demo-plugin-0.1.0.tar.gz
+ * Copy to : ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.9.0.tar.gz
+ */
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Demo plugin - Webhook events', () => {
+    const pluginId = 'com.mattermost.demo-plugin';
+    const pluginName = 'com.mattermost.demo-plugin-0.9.0.tar.gz';
+
+    let testTeam;
+    let testUser;
+    let admin;
+    let testChannel;
+    let newTeam;
+
+    before(() => {
+        // # Set plugin settings
+        const newSettings = {
+            PluginSettings: {
+                Enable: true,
+                RequirePluginSignature: false,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        cy.apiInitSetup().then(({team, user}) => {
+            admin = user;
+            testTeam = team;
+
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+
+            // # Create team before installing demo plugin
+            cy.apiCreateTeam('team', 'Team').then(({team: anotherTeam}) => {
+                newTeam = anotherTeam;
+            });
+
+            // # Uninstall all plugins
+            cy.apiGetAllPlugins().then(({plugins}) => {
+                const {active, inactive} = plugins;
+                inactive.forEach((plugin) => cy.apiRemovePluginById(plugin.id));
+                active.forEach((plugin) => cy.apiRemovePluginById(plugin.id));
+            });
+
+            // # Install demo plugin
+            cy.apiUploadPlugin(pluginName).then(() => {
+                cy.apiEnablePluginById(pluginId);
+            });
+
+            // # Enable webhook events
+            cy.postMessage('/demo_plugin true');
+            cy.wait(TIMEOUTS.TWO_SEC);
+
+            // * Verify that hooks are enabled
+            cy.getLastPost().contains('enabled', {matchCase: false});
```

Comment:
Nit pick: It seems more idiomatic to use `.should('contain')`.

Reply:
`should` is not working with the matchCase, but others should be changed.